src/ide/spans.ts

```diff
@@ -649,7 +649,7 @@ export class CodeBlockSpan extends BlockSpan {
 
   disable() {
     if(!this.disabled) {
-      this.source.info = "eve disabled";
+      this.source.info = (this.source.info ? this.source.info : "eve") + " disabled";
```

Comment:
With the `syntax()` guy this logic becomes a little cleaner in `enable()` and `disable()` since it no longer requires a `replace()`